package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "movieapp",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
```

Comment:
ah, cool. so you tried some testing stuff!

small note, this should go in `devDependencies`. when you npm install, just do a `npm install <lib> --save-dev`.